### PR TITLE
Bugfix: `Any` and `Object` default mapping

### PR DESCRIPTION
**Pull request by znicholasbrown**
This PR changes the way defaults are handled for `any` and `object` properties; previously unmapped defaults were returned when a nullish value was passed to the service. Now, defaults are mapped the same as other response values

### CODE REVIEW

File: src/services/schemas/utilities.ts [hunk 1]
```diff
@@ -52,17 +52,17 @@ export function getSchemaRequestValue(schema: Schema, values: SchemaValues): Sch
 /*
  * Gets a UI friendly version of a property's api friendly value. Used for mapping.
  */
-export function getSchemaPropertyResponseValue(property: SchemaProperty, value: SchemaValue): SchemaValue {
-  const service = schemaPropertyServiceFactory(property, 0)
+export function getSchemaPropertyResponseValue(property: SchemaProperty, value: SchemaValue, level?: number): SchemaValue {
+  const service = schemaPropertyServiceFactory(property, level ?? 0)
```
Comment (pleek91) [inline src/services/schemas/utilities.ts:56]:
```suggestion
export function getSchemaPropertyResponseValue(property: SchemaProperty, value: SchemaValue, level: number = 0): SchemaValue {
  const service = schemaPropertyServiceFactory(property, level)
```

File: src/services/schemas/utilities.ts [hunk 2]
```diff
@@ -52,17 +52,17 @@ export function getSchemaRequestValue(schema: Schema, values: SchemaValues): Sch
...
   return service.mapResponseValue(value)
 }
 
 /*
  * Gets a api friendly version of a property's UI friendly value. Used for mapping.
  */
-export function getSchemaPropertyRequestValue(property: SchemaProperty, value: SchemaValue): SchemaValue {
-  const service = schemaPropertyServiceFactory(property, 0)
+export function getSchemaPropertyRequestValue(property: SchemaProperty, value: SchemaValue, level?: number): SchemaValue {
+  const service = schemaPropertyServiceFactory(property, level ?? 0)
```
Comment (pleek91) [inline src/services/schemas/utilities.ts:65]:
```suggestion
export function getSchemaPropertyRequestValue(property: SchemaProperty, value: SchemaValue, level: number = 0): SchemaValue {
  const service = schemaPropertyServiceFactory(property, level)
```

**Review by pleek91 — COMMENTED** [review]
Syntax suggestion for the utility changes

**Review by pleek91 — APPROVED** [review]

File: src/services/schemas/utilities.ts (hunk 1 above)
Comment (znicholasbrown) [reply src/services/schemas/utilities.ts]: For some reason I thought TS didn't respect default values 😮‍💨  nice suggestion

**Pull request merged**